models/_sector/dex/trades/scroll/platforms/nuri_scroll_base_trades.sql

```diff
@@ -0,0 +1,21 @@
+{{
+    config(
+        schema = 'nuri_scroll',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    uniswap_compatible_v3_trades(
+        blockchain = 'scroll',
+        project = 'nuri',
```

Comment:
Hi @Jam516,
we may need to add `nuri` into this `dex.info`:
https://github.com/duneanalytics/spellbook/blob/main/models/dex/dex_info.sql

Reply:
Thanks! Added it now